spec/RoomListSpec.js

```diff
@@ -8,7 +8,7 @@ describe('RoomList', function() {
       roomList,
       props;
   beforeEach(done => {
-    props = {};
+    props = {roomlist: ['ThaShizznit']};
```

Comment:
we might want to enforce consistency on whether we put a space before & after curly braces (i.e { someObject }) or no